app/models/spree_avatax_certified/line.rb

```diff
@@ -1,21 +1,20 @@
 module SpreeAvataxCertified
   class Line
-    attr_reader :order, :invoice_type, :lines, :stock_locations, :return_authorization
+    attr_reader :order, :invoice_type, :lines
 
     def initialize(order, invoice_type, return_authorization=nil)
-      @logger ||= AvataxHelper::AvataxLog.new('avalara_order_lines', 'SpreeAvataxCertified::Line', 'building lines')
+      @logger ||= AvataxHelper::AvataxLog.new('avalara_order_lines', 'SpreeAvataxCertified::Line', "Building Lines for Order#: #{order.number}")
       @order = order
       @invoice_type = invoice_type
       @lines = []
       @return_authorization = return_authorization
       @stock_locations = order_stock_locations
       build_lines
+      @logger.debug @lines
     end
```

Comment:
Again, better to access `logger`, as a method. That well, it'll be better isolated, than having it mixed up here in the initialize